io/FV3GFS_io.F90

```diff
@@ -1114,8 +1107,12 @@ subroutine sfc_prop_restart_read (Sfcprop, Atm_block, Model, fv_domain, warm_sta
           if (Model%rdlai) then
             Sfcprop(nb)%xlaixy(ix)        = sfc_var2(i,j,nvar_s2m+29)
           endif
-        else if (Model%lsm == Model%lsm_ruc .and. Model%rdlai) then
-          Sfcprop(nb)%xlaixy(ix) = sfc_var2(i,j,nvar_s2m+19)
+        else if (Model%lsm == Model%lsm_ruc) then
+          ! Initialize RUC snow cover on ice from snow cover
+          Sfcprop(nb)%sncovr_ice(ix)      = Sfcprop(nb)%sncovr(ix)
```

Comment:
Does this mean there will be no snow cover on ice in cold start ICs?

Reply:
There is none, correct. This is a RUC LSM only field, and RUC LSM coldstarts from Noah LSM (or whatever the LSM in the official GFS is). If that changes at some point, we can update the logic here.

Reply:
Thanks. Also, will GFS_phys_time_vary_init be called only once in CCPP or every physics time step? I am curious as I see the sncovr_ice is set to snow cover again for lsm_ruc in GFS_phys_time_vary_init. Will sncovr_ice become all negative during integration?

Reply:
`GFS_phys_time_vary_init` is only called once as part of the call to ccpp-physics init, the negative values are created by reading a non-existent variable in `FV3GFS_io.F90` into `sncovr` (and then copying it into `sncovr_ice`).

Reply:
Thanks for the explanation.